docs/metasploit-framework.wiki/Metasploit-Guide-LDAP.md

```diff
@@ -56,6 +56,7 @@ CN=Administrator CN=Users DC=domain DC=local
                      | CN=Enterprise Admins,CN=Users,DC=domain,DC=local || CN=Schema Admins,CN=Users,DC=domain,DC=local || CN=Adm
                      inistrators,CN=Builtin,DC=domain,DC=local
  name                Administrator
+ objectsid           S-1-5-21-3402587289-1488798532-3618296993-500
```

Comment:
The section below talks about the various actions that we have available, but this list isn't updated to reflect the new additions you added to the YAML file.

Reply:
Fixed this n pushed up changes.